### PR TITLE
Add state machine for tasks

### PR DESCRIPTION
**Pull request by alXmas**

### CODE REVIEW

**Comment by alXmas** [comment]
Fix

File: app/models/task.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Task < ApplicationRecord
-  include State
+  include TaskStateMachine
```
Comment (artempartos) [inline app/models/task.rb]: include TaskStateMachine -> include StateMachines::TaskStateMachine
isn't it?

Comment (alXmas) [reply app/models/task.rb]: Yeah, fix it

**Comment by artempartos** [comment]
👍

**Pull request merged**